Geometry/Cells/Tests/unionCell_test.f90

```diff
@@ -0,0 +1,166 @@
+module unionCell_test
+
+  use numPrecision
+  use dictionary_class,   only : dictionary
+  use dictParser_func,    only : charToDict
+  use surfaceShelf_class, only : surfaceShelf
+  use unionCell_class,    only : unionCell
+  use funit
+
+  implicit none
+
+  ! Parameters
+  character(*), parameter :: SURF_DEF = "&
+  & surf1 { id 13; type sphere; origin (0.0 0.0 0.0); radius 5.0;} &
+  & surf2 { id 15; type box; origin (0.0 0.0 0.0); halfwidth ( 4.6 4.6 4.6);} &
+  & surf3 { id 4;  type xTruncCylinder; origin (0 0 0); halfwidth 5.1; radius 1.0;} &
+  & surf4 { id 5;  type yTruncCylinder; origin (0 0 0); halfwidth 5.1; radius 1.0;} &
+  & surf5 { id 6;  type zTruncCylinder; origin (0 0 0); halfwidth 5.1; radius 1.0;} "
+
+  ! Note that fill is not really needed to build a cell. It is used by cellShelf only
+  character(*), parameter :: EASYCELL_DEF = "&
+  & id 2; type unionCell; surfaces [-13 -15 ]; filltype outside; "
+  character(*), parameter :: COMPLEXCELL_DEF = "&
+  & id 3; type simpleCell; surfaces [< -13 -15 > # < -4 : -5 : -6 > ]; filltype outside; "
```

Comment:
shouldn't this be ``unionCell`` too?

Reply:
The cell % init procedure doesn't read ``type``, so that could be removed actually.